docs/verification.md

```diff
@@ -0,0 +1,146 @@
+
+# Verification, Validation, and Veracity
+
+This attempts a general, user-facing description of the process whereby a relying party checks whether they will accept VC or VP. An additional goal is to provide a framework and context for the “verification” steps provided in the vc-http-api repository.
+
+This process is broken into three sub-processes called Verification, Validation, and Veracity (“3V”). 
+
+# Table of contents
+
+- [Verification, Validation, and Veracity](#verification-validation-and-veracity)
+- [Definitions](#definitions)
+- [3V Procedure](#3v-procedure)
+  - [Verification](#verification)
+  - [Validation](#validation)
+  - [Veracity](#veracity)
+- [Interfaces between Validation and Verification](#interfaces-between-validation-and-verification)
+- [Sources](#sources)
+
+
+# Definitions
+
+Definitions of the first two terms come from the [VC-DATA-MODEL](https://www.w3.org/TR/vc-data-model). The third is a replacement/repurposing of what the VC-DATA-MODEL calls “Validity”. (Renaming due to insufficient distinction between the words “validation” and “validity”.)
```

Comment:
Putting the reasoning here is fine for the time being, but ultimately, we should fix the VC spec.

Reply:
+1